components/skus/common/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
+  "+third_party/blink/public",
```

Comment:
you are actually including `third_party/blink/public/platform` which is not allowed here

Reply:
fixed